src/render/html/render_how_to_cook.py

```
@@ -0,0 +1,24 @@
+    extra_body = f"""
+        <div class="copyright">
+            <div class="tool-container">
+                <p class="tool-name">How to Cook</p>
+                <p class="tool-version">v1.4.0</p>
+            </div>
+            <p class="generation-info">Generated at {datetime.now().strftime("%Y/%m/%d %H:%M:%S")}.<br>
+                                       Initiated by OBot\'s ACM {Constants.core_version}.<br>
+                                       Wish you everything goes well.</p>
+        </div>
+        """
+    md_html = md_to_html(dish_path, _css_path, extra_body)
```

Ruff (0.8.2)

Local variable `md_html` is assigned to but never used

Remove assignment to unused variable `md_html`

(F841)